src/app/components/dashboard/shared/dashboard-header/dashboard-header.component.scss

```diff
@@ -1,5 +1,7 @@
 $primary: #0082ca;
 
+.activeLink {
+}
```

Comment:
As this part is left empty for the future to implement, maybe it's worth to create an issue for it so we don't forget about it?